### ISSUE TITLE
New check for `addr:city`

### ISSUE THREAD

**Issue by Dimitar5555** [osm-fr/osmose-backend#1353]
A new check for `addr:city` would be nice. The easiest way to set it up (IMO) would be to get all `name`s of `place=hamlet/village/town/city` for a specific region/country and check each `addr:city` tag in that area. If it's not on the list, raise an issue.

**Comment by frodrigo** [comment]
It exists already
https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_osmosis_relation_associatedStreet.py#L646

But look for admin boundary. Probably not adapted to all countries. So it is only enabled per country.

https://github.com/osm-fr/osmose-backend/blob/master/osmose_config.py#L273

Can be enabled for your area.

**Comment by Dimitar5555** [comment]
All municipalities' relations should be mapped in Bulgaria but the cities/villages admin relations are being imported currently. Would that be a problem?

**Comment by frodrigo** [comment]
> All municipalities' relations should be mapped in Bulgaria but the cities/villages admin relations are being imported currently. Would that be a problem?

Yes, if Bulgarian `addr:city` should match cities/villages and not municipalities.

**Comment by Dimitar5555** [comment]
It's going to be on hold for some time then. I will contact the person who is importing them to check if they are done.
